vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -380,6 +380,10 @@ - (void)fetchPublisherActivityFromURL:(NSURL *)URL
                         publisherBlob:(nullable NSString *)publisherBlob
                                 tabId:(uint64_t)tabId
 {
+  if (!URL.absoluteString) {
```

Comment:
Maybe better to bind these to a temp and pass that to the `parseUrl` function instead but this is optional.

Reply:
Not sure I'm following, bind these to a temp?